### PR TITLE
14.0 mig sale invoice policy

### PR DESCRIPTION
**Pull request by kimirondon**

### CODE REVIEW

File: sale_invoice_policy/models/product_template.py [hunk 1]
```diff
@@ -28,6 +27,14 @@ def _default_default_invoice_policy(self):
         inverse="_inverse_invoice_policy",
     )
 
+    @api.model
+    def default_get(self, fields_list):
```
Comment (rousseldenis) [inline sale_invoice_policy/models/product_template.py:31]: @kimirondon Why does the default lambda not work ?

**Comment by rousseldenis** [comment]
Runbot is complaining about these : 

image


Could you check ?

File: sale_invoice_policy/models/product_template.py (hunk 1 above)
Comment (kimirondon) [reply sale_invoice_policy/models/product_template.py:31]: Hello @rousseldenis, sorry for my lateness!, about the default, i had an issue with the method _reflect_selections,
but i believe #1762  will be merge soon.

Comment (rousseldenis) [reply sale_invoice_policy/models/product_template.py:31]: @kimirondon If you want to review that one and close this.

**Pull request closed without merging**